package.json

```diff
@@ -13,11 +13,16 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "typescript": "^2.8.0",
+    "apollo-boost": "^0.4.7",
+    "graphql": "^14.6.0",
     "react": "^16.13.1",
+    "react-apollo": "^3.1.3",
     "react-dom": "^16.13.1",
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.1",
-    "serve": "^11.3.0"
+    "serve": "^11.3.0",
+    "typescript": "^2.8.0"
```

Comment:
I noticed we have typescript here - will we be using typescript to build our components?

Reply:
not sure how it sneaked in here but i'm considering it - i don't think anyone on our team has much experience which is what's holding me back